api/swagger/swaggerDoc.yaml

```diff
@@ -290,6 +383,45 @@ definitions:
       detail:
         type: object
         example: {}
+  400ErrorResponseUsernameV03:
+    properties:
+      status:
+        type: number
+        example: 400
+      type:
+        type: string
+        example: invalid-username
+      message:
+        type: object
+        example: 'Usernames should be 5-25 characters, letters or numbers only. Please revise your username.'
```

Comment:
How come this is in quotes?  (The other message strings aren't.)

Reply:
That's interesting. I don't know. I work with the json file (since I prefer json) and make the Swagger editor convert it to yaml.